src/main/java/uk/gov/register/presentation/representations/YamlWriter.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.register.presentation.representations;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import io.dropwizard.jackson.Jackson;
+import io.dropwizard.views.View;
+import org.jvnet.hk2.annotations.Service;
+import uk.gov.register.presentation.view.ListResultView;
+import uk.gov.register.presentation.view.SingleResultView;
+
+import javax.inject.Inject;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.MessageBodyWriter;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+@Produces(ExtraMediaType.TEXT_YAML)
+@Service
+public class YamlWriter implements MessageBodyWriter<View> {
```

Comment:
I tried that, but RepresentationWriter assumes that a single element serializes the same as a list of one element, which isn't true in YAML :/